### PR TITLE
Adds support for the --asset-name flag when cutting a plugin

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
This PR adds support for an `--asset-name` flag in the `cutplugin` command, to support to reference a given asset in the release to sign instead of forcing for only one tarball to be present. The use case that triggered this need is the focalboard release, where we want to include several tarball assets in the same release, and only add the signature for the plugin one.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -449,7 +451,7 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 	msg := fmt.Sprintf("@%s triggered a plugin %srelease process using `%s`.\nTag %s created in`%s`. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", slashCommand.Username, releasePrefix, command, tag, repo)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
 		if !force {
-			WriteErrorResponse(w, NewError(fmt.Sprintf("@%s Tag %s already exists in %s. Not generating any artifacts. Use --force to regenerate artifacts.", slashCommand.Username, repo, tag), nil))
+			WriteErrorResponse(w, NewError(fmt.Sprintf("@%s Tag %s already exists in %s. Not generating any artifacts. Use --force to regenerate artifacts.", slashCommand.Username, tag, repo), nil))
```
Comment (mgdelacroix) [inline server/server.go:454]: This change is unrelated, just noticed it when using the bot and included it here

**Review by cpanato — APPROVED** [review]

**Review by jespino — APPROVED** [review]
LGTM

**Review by cpoile — APPROVED** [review]
Awesome!
